Libraries/LibJS/Cell.cpp

```diff
@@ -40,7 +40,7 @@ void Cell::Visitor::visit(Value value)
         visit(value.as_cell());
 }
 
-Heap& Cell::heap()
+Heap& Cell::heap() const
```

Comment:
I have a feeling like this isn't ok, is this ok?

Reply:
I think it's ok.